nubia/internal/plugin_interface.py

```diff
@@ -136,3 +136,10 @@ def getBlacklistPlugin(self):
         Any return other then 0 will block command execution
         """
         return CommandBlacklist()
+
+    def get_prompt_tokens(self):
```

Comment:
We'd really appreciate a type annotation for this function and its call site. Even though Nubia itself is not fully type annotated yet, it's our goal to eventually leverage the type checking, so it would be nice for all the new code to conform :)